src/components/DropDown/DropDown.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './DropDown.css';
+const DropDown = () => {
```

Comment:
pass a function to get the value of the clicked item like this
```suggestion
const DropDown = ({ handleSelectedCategory }) => {
```